src/lyexport.py

```diff
@@ -1,10 +1,11 @@
 
-from mingus.extra.lilypond import to_png
+from mingus.extra.lilypond import to_png, to_pdf
 from tempfile import TemporaryDirectory
 from threading import get_ident
 import matplotlib
 import matplotlib.pyplot as plt
 import matplotlib.image as mpimg
```

Comment:
Let's leave matplotlib as a dependency for now while we switch to PIL

Reply:
ok